inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/service/ConceptLinkingServiceImpl.java

```diff
@@ -72,7 +72,7 @@
 public class ConceptLinkingServiceImpl
     implements InitializingBean, ConceptLinkingService
 {
-    private final Logger log = LoggerFactory.getLogger(getClass());
+	private final Logger log = LoggerFactory.getLogger(getClass());
```

Comment:
Please avoid unnecessary whitespace changes - in particular ones which checkstyle will complain about :)